src/validations/actionSchema.ts

```diff
@@ -5,52 +5,30 @@ export const actionSchema = Yup.object().shape({
   feedback: Yup.string().required('Please fill out email')
 });
 
-export const lifecycleIdSchema = Yup.object().shape({
-  expirationDateDay: Yup.string()
-    .trim()
-    .length(2)
-    .required('Please include a day'),
-  expirationDateMonth: Yup.string()
-    .trim()
-    .length(2)
+const sharedValidDateSchema = Yup.object().shape({
```

Comment:
@DylanSpOddball  I was able to pull out the `validDate` schema and DRY up the code.